src/const.js

```diff
@@ -18,4 +18,9 @@ const FilterType = {
   PAST: 'past'
 };
 
-export { DATE_FORMAT, EVENT_COUNT, FilterType };
+const Mode = {
+  DEFAULT: 'default',
+  EDITABLE: 'editable'
```

Comment:
Так, кажется, понятнее будет что там происходит

```suggestion
  VIEW: 'view',
  EDIT: 'edit'
```